index.html

```diff
@@ -16,16 +17,249 @@
 </head>
 
 <body>
+  <header>
+    <div class="jumbotron">
+      <div class="row">
+        <div class="col-12">
+          <h1 class="d-flex justify-content-center">Space Walkers</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-12">
+
+          <nav class="nav d-flex justify-content-center">
+
+            <a class="nav-link active" href="#">Home</a>
+            <a class="nav-link" href="#">About Us</a>
+            <a class="nav-link" href="#">Blog</a>
+            <a class="nav-link" href="#">Contact</a>
+            <a class="nav-link" href="#">Sign In</a>
+          </nav>
+        </div>
+        <!-- col-sm-12 -->
+      </div>
+      <!-- row -->
+    </div>
+    </div>
+    <!-- jumbotron -->
+  </header>
   <div class="container">
+    <!-- carousel -->
+    <div class="row row-no-margin d-flex justify-content-center mb-5">
+      <div class="col-10">
+        <div id="carouselExampleControls" class="carousel slide mt-5" data-ride="carousel">
```

Comment:
If you look at your carousel, the prev and next arrows are not quite aligned in the center. If you want the `mt-5`, you can remove it on line 50 and apply it to the line above.